src/main/java/org/broadinstitute/hellbender/utils/io/IOUtils.java

```diff
@@ -589,4 +570,25 @@ public static void assertFileIsReadable(final Path path) {
             throw new UserException.CouldNotReadInputFile(path, cloudBoom.getCode() + ": " + cloudBoom.getMessage(), cloudBoom);
         }
     }
+
+    /**
+     * Checks that one or more user provided files are in fact regular (i.e. not a directory or a special device) readable files.
+     *
+     * @param files the input files to test.
+     * @throws IllegalArgumentException if any input file {@code file} is {@code null} or {@code files} is {@code null}.
+     * @throws UserException if any {@code file} is not a regular file or it cannot be read.
+     */
+    public static void canRead(final File... files) {
```

Comment:
I think `IOUtils.fileIsReadable()` is fine and not too verbose (slightly less verbose than the name Louis objected to). `canRead()` is just confusing, since the method tests file vs. non-file in addition to readability.